Vagrantfile

```diff
@@ -18,8 +18,8 @@ Vagrant.configure("2") do |config|
   config.vm.provider :virtualbox do |vb|
     vb.customize [
       "modifyvm", :id,
-      "--memory", "1024",
-      "--cpus", "2",
+      "--memory", "4096",
+      "--cpus", "4",
```

Comment:
Halved the time for me, being able to use 4 cores on my laptop was nice. Meant I couldn't do any work, but I just let it run over lunch.